public/data/db.json

```diff
@@ -0,0 +1,512 @@
+{
```

Comment:
Duplicate file it is supposed to be moved in src not in public -5

Reply:
bug fixed.